api/src/controllers/homepage.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * This module contains handlers for the "homepage" route.
+ * @module homepage
+ */
+const Homepage = require('../models/homepage.model');
+const { fillErrorObject } = require('../middleware/error');
+
+/**
+ * Get homepage content on /homepage/:team_id
+ * @param {*} req
+ * @param {*} res
+ * @returns
```

Comment:
Could you fill out the rest of the docs, perhaps using other controller docs as a reference.

Reply:
done